city_scrapers/spiders/pitt_housing.py

```diff
@@ -0,0 +1,145 @@
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+
+        for item in self._create_meetings(response): 
+
+            data = {
+                '_type': 'event',
+                'name': 'Housing Authority of the City of Pittsburgh',
+                'event_description': 'Board of Commissioners meeting',
+                'classification': BOARD,
+                'start': self._parse_start(item),
+                'end': {'date': None, 'time': None, 'note': None},
+                'all_day': False,
+                'location': {
+                    'address': 'Civic Building, 200 Ross St., Pittsburgh, PA, 15219',
+                    'name': '',
+                    'neighborhood': ''
+                },
+                'documents': self._parse_documents(item),
+                'sources': self._parse_sources(item),
+            }
+
+            data['status'] = self._generate_status(data, text='')
+            data['id'] = self._generate_id(data)
+
+            yield data
+
+    def _create_meetings(self, response):
+        """
+        Returns a list containing lists corresponding to each meeting.
+        """
+        meetings = []
+        parser = MyHTMLParser()
+        meetings_xpath = '//div[@id="the-copy"]/div[2]/p[contains(strong, "2018 HACP Board Agendas")]/following-sibling::p'
+
+        for item in response.xpath(meetings_xpath).extract():  
+            parser.feed(item)
+            meeting = [parser.date, [parser.doc_url]]
+
+            # If meetings is empty, append meeting
+            if not meetings:
+                meetings.append(meeting)
+
+            # If meetings is not empty, get the last meeting date and compare 
+            # it to parser.date. Append parser.doc_url to last meeting if 
+            # parser.date == the last meeting date. Otherwise, append the whole
+            # meeting to meetings.
+            else:
+                last_meeting = meetings[-1]
+                last_meeting_date = last_meeting[0]                
+                if parser.date == last_meeting_date:
+                    if parser.doc_url:
+                        last_meeting[1].append(parser.doc_url)  
+                else: 
+                    meetings.append(meeting)
+
+        return meetings
+
+    def _parse_start(self, item):
+        """
+        Parse start date and time.
+        """
+
+        date_string = item[0]
+        try:
+            dt = dtparse(date_string)
+            return {'date': dt.date(), 'time': None, 'note': None}
+        except ValueError:
+            return {'date': None, 'time': None, 'note': None}
+
+    def _parse_documents(self, item):
+        """
+        Parse or generate documents.
+        """
+
+        # Create documents, which will be a list of dictionaries corresponding 
+        # to each agenda or minutes document found.
+        documents = []
+        for doc in item[1]: 
+            if re.search(r'Agenda',doc):
```

Comment:
Since you aren't using a regular expression here, something like `agenda in doc.lower()` might be more clear. You should be able to remove `re` in this file generally since it looks like it's only used for string lookups